content/book_phone.md

```diff
@@ -4,12 +4,13 @@ tags:
   - appointments
   - booking
 ---
-
 # How to book a phone appointment
 
-Phone **0800 138 3944**.
+[Book online](/telephone_appointments/new){: .button #phone-button style="font-size: 0.9em; padding-left: 2em; padding-right: 2em;"}
```

Comment:
Can you make this `/telephone-appointments/new` so we avoid a redirect? Also in a separate commit could you remove the redirect from the routes?